### ISSUE TITLE
Improving performance of javascript

### ISSUE THREAD

**Issue by imdhruvgupta** [phpmyadmin/phpmyadmin#15168]
[Migrating from javascript to jQuery]

I have observed that although most of the javascript code is written in jQuery, but in few js files instead of using jQuery selectors, elements are accessed by document.getElementById. Apart from that I have observed that few files are missing file overview comments. (Bad coding practice)

**Comment by imdhruvgupta** [comment]
I am working on it. Will create a PR soon.

**Comment by williamdes** [comment]
> I am working on it. Will create a PR soon.

Great !


@imdhruvgupta Do not forget to create the PR from and against master ;)
See: #15165 (if you modify designer files)

**Comment by kartik1000** [comment]
@imdhruvgupta it would be a great idea to share info. about those files so that others can help you out too. what do you think @williamdes ?

**Comment by williamdes** [comment]
@kartik1000 Good idea, posting a to-do list to this issue and posting updates when taking files to work on and when some files are done

**Comment by mauriciofauth** [comment]
I'm not really convinced in converting every plain JavaScript to jQuery. JS evolved a lot in recent years and converting everything to jQuery is not a huge benefit anymore.

**Comment by imdhruvgupta** [comment]
> I'm not really convinced in converting every plain JavaScript to jQuery. JS evolved a lot in recent years and converting everything to jQuery is not a huge benefit anymore.

Yes, that's why I did not opened a pull requests following the conversation in the issue #15165 .I think jQuery still has a lot of benefits as compared to vanilla js but still particularly jQuery selectors does not add up to a lot of performance gain.
